packages/react/src/components/Card/Card.js

```diff
@@ -19,6 +19,7 @@ import { LinkWithIcon } from '../LinkWithIcon';
 import PropTypes from 'prop-types';
 import React from 'react';
 import settings from 'carbon-components/es/globals/js/settings';
+import TableOfContents from '../TableOfContents/TableOfContents';
```

Comment:
@greeshmareddyg this might have been a mistake to add `TableofContents` here?

Reply:
yeah, updated Jeff.